docs/send-data/collect-from-other-data-sources/azure-blob-storage/troubleshoot-log-collection.md

```diff
@@ -1,12 +1,13 @@
 ---
-id: troubleshoot-azure-blob-storage-log-collection
-title: Troubleshoot Azure Blob Storage Log Collection
+id: troubleshoot-log-collection
+title: Troubleshoot Azure Storage Log Collection
 sidebar_label: Troubleshooting
 description: Follow these steps to learn why log data is not flowing into Sumo from Azure Blob Storage.
 ---
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-If logs do not start flowing into Sumo Logic after you perform the [Collect Logs from Azure Blob Storage](collect-logs-azure-blob-storage.md) procedure, see the troubleshooting tips below.
+If logs do not start flowing into Sumo Logic after you perform the Collect Logs from Azure Blob Storage procedure, see the troubleshooting tips below.
 
 ## Error while deploying the ARM template
 
```

Comment:
In this section, the link for "Refer to this video" doesn't work.

Reply:
removed the link for now, I couldn't find it.